src/views/Jobs/JobResultsTable.vue

```diff
@@ -708,6 +751,98 @@ export default Vue.extend({
         this.pagination.sortBy = column;
         this.pagination.descending = false;
       }
+    },
+    visualize() {
+      const predictions = this.selected.map(jobPrediction => {
+        const addedReactionIds = [
+          ...jobPrediction.heterologous_reactions,
+          ...jobPrediction.synthetic_reactions
+        ];
+        const promises = this.getAddedReactions(addedReactionIds);
+        Promise.all(promises).then(addedReactions => {
+          const card = {
+            uuid: uuidv4(),
+            name: `Job #${this.prediction.id}`,
+            organism: this.organism,
+            modelId: this.model.id,
+            method: "pfba", // TODO - should this be default?
+            dataDriven: false,
+            // Design card fields
+            objective: {
+              reaction: null,
+              maximize: true
+            },
+            reactionAdditions: addedReactions,
+            reactionKnockouts: [],
+            geneKnockouts: [],
```

Comment:
Now I realized that we missed the knockout part. We also have knockouts in the prediction and if method is opt_gene than those are gene knockouts, otherwise - reaction